client/src/pages/private/EOIViewDetails.js

```diff
@@ -1,19 +1,17 @@
 import React, { useEffect, useState } from 'react';
 import Grid from '@material-ui/core/Grid';
-import { Routes } from '../../constants';
-import { useLocation, Redirect } from 'react-router-dom';
 import { Page, CheckPermissions } from '../../components/generic';
 import { Form } from '../../components/employer-form';
 import { scrollUp } from '../../utils';
 import store from 'store';
 
-export default () => {
+export default ({ match }) => {
```

Comment:
Since you are using only `match.params.id` would be better to just pass the id itself as a prop

Reply:
This is the syntax that's provided through Routes. I don't know if/how I can pass id individually.

Reply:
Ah, I see. You are not responsible for rendering the component. Nvm!